agent/agentssh/agentssh.go

```diff
@@ -1044,6 +1055,11 @@ func (s *Server) trackSession(ss ssh.Session, add bool) (ok bool) {
 // Close the server and all active connections. Server can be re-used
 // after Close is done.
 func (s *Server) Close() error {
+	return s.close(context.Background())
```

Comment:
Do we need this extra method with the context argument? I don't see it used elsewhere.

Reply:
Originally I intended to use `s.close` in `Shutdown` too, but I oopsied that change. I'll revert this though since ctx is only used for logger and we don't even utilize that feature often.